launch-game.sh

```diff
@@ -1,9 +1,36 @@
 #!/bin/sh
-# Note: this relies on the non-standard -f flag implemented by gnu readlink
-mono OpenRA.Game.exe Engine.LaunchPath="$(readlink -f $0)" "$@"
-if [ $? != 0 -a $? != 1 ]
+MODLAUNCHER=$(python -c "import os; print(os.path.realpath('$0'))")
+
+# Prompt for a mod to launch if one is not already specified
+MODARG=''
+if [ z"${*#*Game.Mod}" = z"$*" ]
 then
-	ZENITY=`which zenity` || echo "OpenRA needs zenity installed to display a graphical error dialog. See ~/.openra. for log files."
-	$ZENITY --question --title "OpenRA" --text "OpenRA has encountered a fatal error.\nLog Files are available in ~/.openra." --ok-label "Quit" --cancel-label "View FAQ" || xdg-open https://github.com/OpenRA/OpenRA/wiki/FAQ
+	if which zenity
+	then
+		TITLE=$(zenity --forms --add-combo="" --combo-values="Red Alert|Tiberian Dawn|Dune 2000|Tiberian Sun" --text "Select mod" --title="" || echo "cancel")
```

Comment:
It doesn't and isn't supposed to.  That is [launch-mod.sh](https://github.com/OpenRA/OpenRAModTemplate/blob/master/launch-mod.sh)'s job.